src/components/Input/Input.js

```diff
@@ -81,6 +81,18 @@ const containerNoMarginStyles = ({ noMargin }) =>
     margin-bottom: 0;
   `;
 
+const textAlignLeftStyles = ({ textAlign }) =>
+  textAlign === 'left' &&
+  css`
+    text-align: left;
+  `;
+
+const textAlignRightStyles = ({ textAlign }) =>
+  textAlign === 'right' &&
+  css`
+    text-align: right;
+  `;
```

Comment:
Hey Ju - thanks for the PR!

I would suggest to you that for your CSS classes, embed the default in the main styles instead of having two separate styles. Because that way, when we have a CSS-only version, people don't need to add an extra modifier to get the default behavior.

So, just take your `text-align: left` into the default styles. Maybe it's not even needed.

The other thing to think about is the CSS label. I think you may need to include something like this:

```javascript
const alignRightStyles = ({ textAlign }) => textAlign === 'right' && css`
  label: input__container--right;
  text-align: right;
`;
```

But I think @connor-baer and @felixjung can clarify.